lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,71 @@
+    <h3>Stealth power.</h3>
+    <p>StarBook Mk V looks unassuming in matte black anodized aluminum, but it’s engineered to pack a punch. 11th-gen Intel Core processors deliver performance on demand while sipping power. Low-latency memory keeps everything snappy. And the over-provisioned solid state StarDrive offers incredible speeds for opening your apps and files.</p>
+  </section>
+
+  <section id="display">
+    <img alt="StarBook Mk V open 180 degrees" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hinge.png") %>" />
+    <div>
+      <h3>Matte or glossy—why not both?</h3>
+      <p>The 14-inch ARC display features a protective hard coating that lets color shine through like glass while diffusing ambient light like a matte display. Combined with IPS technology and 400 nits of brightness, the display is visible in nearly any light and angle. Which is convenient, because the 180° hinge lets you get some pretty wild angles.</p>
+    </div>
+  </section>
+
+  <section id="trackpad">
+    <div>
+      <h3>Slip and swipe.</h3>
+      <p>Glide across the large glass trackpad for precise and smooth multi-touch gestures. Scroll through documents or web pages, fly through your workspaces, and swipe to go back throughout elementary OS—all with 1:1 gestures that perfectly track your fingers.</p>
+    </div>
+    <figure>
+      <span>5.5&Prime;</span>
+    </figure>
+  </section>
+
+  <section id="keyboard">
+  <img alt="StarBook Mk V top-down with keyboard" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/keyboard-us.png") %>" />
+  <div>
+    <h3>A keyboard that’s pure function.</h3>
+    <p>Choose a US or UK layout featuring comfortable backlit keys with snappy scissor mechanisms. Tap a single key to switch between F1 through F12 or function keys that control volume, media, brightness, and more. Plus, an extra column of handy keys including Home, End, and Page Up/Down.</p>
+  </div>
+  </section>
+
+  <section id="privacy">
+    <img alt="StarBook Mk V webcam with illuminated privacy light" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/privacy.png") %>" />
+    <hr />
+    <h3>Ready for remote, with privacy built-in.</h3>
+    <p>The HD webcam is flanked by stereo microphones for clear video and voice calls. The bright white indicator shines any time the camera is on, while elementary OS lets you know any time the mics are being accessed.</p>
+  </section>
+
+  <section id="connectivity">
+    <img alt="StarBook Mk V side with ports visible" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/side.png") %>" />
+  <h3>Versatile connectivity.</h3>
+  <ul>
+    <li><strong>Wi-Fi 6</strong> and <strong>Bluetooth 5.1</strong> for all things wireless</li>
+    <li><strong>USB-C</strong> with <strong>Thunderbolt 4</strong>, power delivery, and display support for charging, accessories, and output all in one
```

Comment:
I have a few questions about this line:

* Could `power delivery` just be `charging`?
* What does `display support for charging` mean?
* What is `output`?

Reply:
USB Power Delivery is A Thing:tm: that specifically means it can negotiate power from various wattage chargers. We cna just say "charging" but it doesn't convey the same information.